src/lib/components/RangeSlider.svelte

```diff
@@ -217,6 +227,8 @@
       value = [value[0], round(newValue, stepDecimals)];
       lastMoved = 'end';
     }
+    // Dispatch change event
+    dispatch('change', [value]);
```

Comment:
1. This will only dispatch on mouse click, but the value can be updated multiple ways (dragging the thumb (values), keyboard left/right, etc.  Could probably just be a reactive statement (`$: dispatch('change', value)`).  See [Form](https://github.com/techniq/svelte-ux/blob/bd7aa07f28c43b5b07a3a7c0ea9be89564c361cc/src/lib/components/Form.svelte#L19) and [NumberStepper](https://github.com/techniq/svelte-ux/blob/bd7aa07f28c43b5b07a3a7c0ea9be89564c361cc/src/lib/components/NumberStepper.svelte#L18) for a few examples.
2. Any reason `value` is wrapped in an array (it is already an array)?

Reply:
updated into a reactive statement which seems to work great!

The only downside is that it now fires multiple events during sliding it instead of one single upon setting the value, but it does make the keyboards work as expected. (Might not be an issue and in fact more correct, seeing as the value actually does incremental changes)